application/modules/admin/controllers/sidebox.php

```diff
@@ -219,7 +219,7 @@ public function save($id = false)
 		// Handle custom sidebox text
 		if($data["type"] == "custom")
 		{
-			$text = $this->input->post("text");
+			$text = $this->input->post("text", false); // Content should be relatively "safe" so we can skip xss_clean.
```

Comment:
I highly doubt the text field from a sidebox is a 'safe' field

Reply:
There is also no such thing as a 'safe' field with user input; sanitize it all because people will try to fuck with everything

Reply:
yea this is the idea. but when it comes to sanitizing tinymce and other wysiwyg editors its sort of hard. what i meant by relatively safe is that the input is within the admin panel so at that point idk what else can be done. but i guess i could try to sanitize the input without using codeigniters xss tho.

Reply:
If it's all within the admin panel it might be okay; but I'd double-check if a user genuinely has 'admin' rights upon posting then. If that's the case, then (personally) I would be okay with dropping the sanitizing

Reply:
that is exactly what i thought but at the same time, i believe this would be better stored in a blob.

Reply:
What is the overall conclusion @mdjong1, is this safe to merge?

Reply:
I can't really say; I haven't done a full-on review of everything but gut feeling says there needs to be extra validation (i.e. user::isAdmin() or whatever we call it) to ensure these fields without xss-filtering aren't abused.

Reply:
i wonder how checking if its admin from within the admin panel will help, but alright :D

Reply:
Because if someone can jack the post request without being logged into the admin panel my brain says they could theoretically post random shit to it

Reply:
I general, I believe it is a very bad idea to remove/skip XSS cleaning regardless. We've seen people attack somewhat insecure AdminCPs before and we should follow those examples with caution.